### ISSUE TITLE
[vacuum_cleaner] Incorrect initial gazebo view

### ISSUE THREAD

**Issue by pariaspe** [JdeRobot/RoboticsAcademy#1915]
Change the initial gazebo view position to a better one. It can be done by adding these lines to the existing worlds:
```xml
<gui fullscreen='0'>
    <camera name='user_camera'>
        <pose frame=''>x y z R P Y</pose>
        <view_controller>orbit</view_controller>
        <projection_type>perspective</projection_type>
    </camera>
</gui>
```

**Comment by GauravKumar9920** [comment]
Hey, I would like to work on this, I will send a pr after I test it...

**Comment by AkashS20** [comment]
Hi, I would like to contribute to this. I'm new to open source so if you could please guide me on solving this,
it will be of great help.

**Comment by pariaspe** [comment]
Open a PR proposing a solution and I'll review it when I have time. You may find useful our [contribution guide](https://github.com/JdeRobot/RoboticsAcademy/blob/master/CONTRIBUTING.md).

**Comment by AkashS20** [comment]
@pariaspe could you elaborate on what `x y z R P Y` should be replaced by? I think it should be replaced by _desired position and orientation of the view in the world,_ but I am unable to understand what desired position and orientation is.

**Comment by jmplaza** [comment]
Hi @AkashS20 ,

no, that is part of the task :-). Use the Basic Vacuum Cleaner exercise on RoboticsAcademy and check the initial position of the camera. That one is not convenient at all, a better camera position would be a bird's eye view. From there the whole apartment and the vacuum cleaner can be seen at a glance. Once you use that exercise you will understand it. 

Cheers,

**Comment by dsg1320** [comment]
Hi, I would like to work on this issue. Could you guide me through solving this since I'm a beginner?

**Comment by pariaspe** [comment]
Hi @dsg1320 , yes please, go ahead and open a PR with your solution. You may find useful our [contribution guide](https://github.com/JdeRobot/RoboticsAcademy/blob/master/CONTRIBUTING.md). The issue is pretty straightforward, but this [link](https://answers.gazebosim.org//question/408/is-there-a-way-to-modify-the-default-camera-position-in-the-world-file/) may help too.

**Comment by GauravKumar9920** [comment]
Is there any preferred view angle ?
cause there's a lot of angles form which I can look at a bot...

**Comment by GauravKumar9920** [comment]
what's the preferred one ?
view -1
![v1](https://user-images.githubusercontent.com/84905312/219849556-e1afbbb1-9c63-4d52-82b6-7e7184b58391.png)
oe view -2
![v2](https://user-images.githubusercontent.com/84905312/219849602-7214b5bf-cb73-421a-8349-d9a08cc51b04.png)

**Comment by pariaspe** [comment]
Hey! What about a top view where it can be seen all the house? Thanks!

**Comment by GauravKumar9920** [comment]
okkayy, I will do it thanks :)

**Comment by pariaspe** [comment]
Solved in #1937 , thanks @GauravKumar9920 !
